docker-compose-build.yaml

```diff
@@ -0,0 +1,39 @@
+---
+version: '3'
+services:
+  orcpub:
+    build:
```

Comment:
Please add `image: orcpub:latest` for better ease of development to allow only rebuilding when desired via a `docker-compose up --build`

Reply:
I assumed we want to have the second compose for dedicated bulding, and the first for dedicated image usage. Of course I can add those, but that will require the `--build` clause. I assumed we want to have the default behaviour to be build on `docker-compose up`